code/iaas/service-discovery/server/src/main/java/io/cattle/platform/servicediscovery/service/impl/RancherImageToComposeFormatter.java

```diff
@@ -0,0 +1,19 @@
+package io.cattle.platform.servicediscovery.service.impl;
+
+import io.cattle.platform.servicediscovery.resource.ServiceDiscoveryConfigItem;
+
+public class RancherImageToComposeFormatter implements RancherConfigToComposeFormatter {
+    private static final String IMAGE_PREFIX = "docker:";
```

Comment:
Is there anything to be concerned here with your integration tests that use sim_contexts and have image uuids that look like `sim:foo:bar`?

You're integration tests do asserts like:
```
assert service.launchConfig.imageUuid == image_uuid
assert container.imageUuid == image_uuid
```

I guess the only issue is simply that you aren't testing this format logic in the integration test, which isnt a big deal